Automattic/simplenote-electron
# Updating webpack and webpack dependent packages.

Fixes an issue with an outdated version of `uglify` used by webpack found here: https://github.com/Automattic/simplenote-electron/pull/743#issuecomment-380208500

**To Test**
* `npm install`
* `npm start`, verify server/app working as expected
* `make package-linux`, verify linux app builds properly.
